ui/schema.graphql

```diff
@@ -0,0 +1,90 @@
+enum AddonSelector {
```

Comment:
Why do we need one more schema file here @vineethvanga18 ? 
`    "relay": "relay-compiler --src ./components --schema ../internal/graphql/schema/schema.graphqls" ` uses the schema in the server itself.

Reply:
It was already present. Though you are right.

Reply:
So... what do we want to do here?

Reply:
Just delete this file and use what is there at 'internal/graphql/schema' directory

Reply:
Call for a re-review when this is done, please.